services/order.py

```diff
@@ -0,0 +1,31 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, MovieSession, Ticket
+
+
+@transaction.atomic
+def create_order(tickets: list[dict], username: str, date: str = None) -> None:
+    user = get_user_model().objects.get(username=username)
+    order = Order.objects.create(user=user)
+    if date:
+        order.created_at = date
+    order.save()
```

Comment:
You are calling the `save()` method on the `order` object immediately after setting the `created_at` date. According to the checklist, you should avoid calling `save()` multiple times if possible. Consider moving the `save()` call to after the loop where tickets are created, if no other modifications to the `order` object are made after this point.